back/src/domain/convention/useCases/broadcast/BroadcastToPoleEmploiOnConventionUpdates.ts

```diff
@@ -29,6 +30,7 @@ export class BroadcastToPoleEmploiOnConventionUpdates extends TransactionalUseCa
     uowPerformer: UnitOfWorkPerformer,
     private poleEmploiGateway: PoleEmploiGateway,
     private timeGateway: TimeGateway,
+    private resyncMode: boolean,
```

Comment:
On peut se le mettre dans un objet, histoire de savoir ce qu'on passe lorsqu'on appelle ?

```suggestion
    private options: { resyncMode: boolean },
```